core/controllers/reader.py

```diff
@@ -456,7 +381,7 @@ class ClassifyHandler(base.BaseHandler):
     REQUIRE_PAYLOAD_CSRF_CHECK = False
 
     @require_playable
-    def post(self, exploration_id):
+    def post(self, unused_exploration_id):
```

Comment:
Is there any reason to keep ``unused_exploration_id`` around? I think the answer is yes (for the future classifier work), in which case please add a TODO to explain why there is an unused parameter here and what it will be used for in the future.